### ISSUE TITLE
Problem with UiBundle javascript, creates white overlay when it shouldn't

### ISSUE THREAD

**Issue by migmolrod** [Sylius/Sylius#9438]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Sylius version   | 1.1.6

There is a bug with the way your admin javascript works. Whenever you hit the "Create" (or "Save changes" if you're editing) button, a semitransparent white overlay with a 'loading' icon (spinner) appears while the request is sent and the new response comes (ie, showing the next page, be it the index page, the update page... whatever you configure when you add the admin routing with the ResourceBundle).

My problem comes when I try to use https://github.com/danielantelo/AddressableBundle within Sylius. This bundle provides a new form type called AddressMapType which consists in a map (you can move the marker to get an address) and a text field with a "Search" button to geocode the search string into an actual address as well as longitude+latitude if you wish instead of using the map itself. Then you can fine-tune the fields for street name, street number, city, country and zip code.

When I click that search button in the AddressMapType, the white overlay + spinner are shown. I can see the AddressMapType is geocoding correctly, finishes its work, but that overlay never goes away.

Tracing the error I've seen that the `form.loadable` selector gets a .loading css class whenever I use any `<button>`. And doing a global search of `.loadable` within the Sylius project, I got where the error is: UiBundle/Resources/private/js/app.js
So, I think we should narrow the selector in UiBundle javascript to only affect the submit buttons, leaving alone plain buttons (non submit). This works for this specific bundle but not sure for others.
The offending line is this:
https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/UiBundle/Resources/private/js/app.js#L30
Changing it with this works for me.
```js
$('form.loadable button[type=submit]').on('click', function() {
      return $(this).closest('form').addClass('loading');
    });
```
But again, not sure if that's enough or if we should try to see other use cases.

**Comment by Zales0123** [comment]
Sound fair for me that for should be "loading" only on submit. However we should take a deeper look into it and investigate is there any place in Sylius Admin panel where such a change could brake something.

**Comment by migmolrod** [comment]
Agree with that.
At the moment, every default Sylius form (I'm using full stack) I've tried is working as intended.

The only other place where that 'loadable' behaviour is used (at least that I've found) are slugs. When you 'unlock' the slug field and edit the name field (in a product or a taxon, the only two that uses it), the slug field correctly gets the `.loading` class and the white overlay + spinner is shown only for that field. So everything seems fine for me like that.

Will be checking out other forms.

**Comment by teohhanhui** [comment]
This bug is still present. Will be fixed by #9627...

**Comment by mamazu** [comment]
It should be done then.

**Comment by teohhanhui** [comment]
This should have been fixed by #9627. Okay to close.
